evm/src/all_stark.rs

```diff
@@ -146,7 +180,30 @@ mod tests {
         keccak_keccak_input_output.extend(Column::singles(
             (0..2 * NUM_INPUTS).map(keccak::registers::reg_output_limb),
         ));
-        let cross_table_lookups = vec![CrossTableLookup::new(
+
+        let cpu_memory_values: Vec<Vec<_>> = (0..4)
```

Comment:
nit: I'd say `cols` or `regs` instead of `values` (likewise for `memory_memory_values`)

Reply:
Let's use `NUM_MEMORY_OPS` vs 4, since we might change it a few times